packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/__snapshots__/ChoiceGroupOption.test.tsx.snap

```diff
@@ -138,14 +138,14 @@ exports[`ChoiceGroupOption renders ChoiceGroup correctly 1`] = `
       className=
           ms-ChoiceField-wrapper
           is-inFocus
-          .ms-Fabric.is-focusVisible & {
+          .ms-Fabric--isFocusVisible & {
```

Comment:
Wondering if we should be using the exported const for this global class name in Utilities instead...

Reply:
Like what we have here?
https://github.com/OfficeDev/office-ui-fabric-react/blob/54e244d8461ec56ad2ffb65984473ce528160e32/packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts#L141-L145

Reply:
yeah, that's the className const. you don't need to add the &:focus part